### PR TITLE
add ToC to publications, move loading of scripts into header

### PR DESCRIPTION
**Pull request by mashehu**
A less markdown based solution compared to #594 
Screenshot 2020-12-10 at 11 26 04

### CODE REVIEW

**Comment by MaxUlysse** [comment]
Love it

**Comment by apeltzer** [comment]
Much better, going to close #594 in favor of this!

**Comment by ewels** [comment]
@mashehu - can you fix this merge conflict please? Would like to merge this in (sorry I've been slow to get to it).

**Comment by maxulysse** [comment]
@mashehu @ewels hopefully merged the confilct,
Can you check if I didn't introduce a bug?

File: includes/header.php [hunk 1]
```diff
@@ -59,41 +59,45 @@
...
-    <script src="/assets/js/moment-timezone-with-data-10-year-range.js"></script>
+      <script src="/assets/js/moment.js"></script>
+      <script src="/assets/js/moment-timezone-with-data-10-year-range.js"></script>
     <?php endif;
-    if(isset($import_leaflet) && $import_leaflet): ?>
-    <link href="/assets/css/leaflet.css" rel="stylesheet">
-    <link href="/assets/css/leaflet.fullscreen.css" rel="stylesheet">
-    <script src="/assets/js/leaflet.js"></script>
-    <script src="/assets/js/Leaflet.fullscreen.min.js"></script>
+    <?php if(isset($import_leaflet) && $import_leaflet): ?>
```
Comment (mashehu) [inline includes/header.php:65]:
```suggestion
    if(isset($import_leaflet) && $import_leaflet): ?>
```
the only error I could find.

**Review by ewels — APPROVED** [review]

**Pull request merged**